Cargo.toml

```diff
@@ -29,6 +29,7 @@ once_cell = "1"
 rand = "0.8"
 tokio = { version = "1", features = ["fs", "time", "rt"] }
 futures = "0.3" # so the doctest for wrap_stream is nice
+pretty_assertions = "1.4.0"
```

Comment:
Unless we are sure we are only relying on features in pretty_assertions that were present in 1.0.0, it's common to add new dependencies at their current versions.

Reply:
@oli-obk ah, yes my mistake. Please revert to 1.4.0. 
@djc Gotcha - I was under the impression that looser deps were better since you automatically got minor bugfixes on rebuild. But I am still a relative Rust noob :)

Reply:
1.4.0 is the same level of loose in this context: Cargo pulls in semver-compatibles by default, any 1.x release with x >= 4 is compatible.

Reply:
TIL thanks!

Reply:
> Unless we are sure we are only relying on features in pretty_assertions that were present in 1.0.0,

I am sure that we are only relying on 1.0 features, and I set the commit message to reflect that.

Reply:
Now that I'm back on a computer, I removed the last commit.